glowing_waffles/photometry.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import print_function, division, absolute_import
+import numpy as np
+import math
+from astropy import units as u
+from astropy.stats import mad_std
+from astropy.io import fits
+from astropy.table import Table, join
+from photutils import aperture_photometry, CircularAperture, CircularAnnulus
+from coordinates import convert_pixel_wcs
```

Comment:
please change the import to `.coordinates`, which will ensure the import happens from within the package.